Python/pdf_tables_extractor/PDFtoCSV.py

```diff
@@ -0,0 +1,20 @@
+import tabula
+
+
+def extract_tables(path, number_pages):
+    tables = tabula.read_pdf(path, multiple_tables=True, pages=number_pages)
+    num = 1
+    if len(tables) != 0:
+        for table in tables:
+            print("Saving file ...")
+            table.to_csv(f'Table-{num}.csv')
+            num += 1
+        print("created a csv file!")
+    else:
+        print("No tables found !")
+
+
+path = input("Enter the path to pdf file")
```

Comment:
Instead of taking CLI inputs, is it possible to pass Argument Parsers to take the input?